Source/Blazorise/Components/Bar/Bar.razor.cs

```diff
@@ -353,5 +394,10 @@ public BarCollapseMode CollapseMode
     /// </summary>
     [CascadingParameter] protected LayoutHeader LayoutHeader { get; set; }
 
+    /// <summary>
+    /// Keeps a single bar item open at a time.
+    /// </summary>
+    [Parameter] public bool ToggleSingle { get; set; }
```

Comment:
How about we have an enum? `BarMenuToggleBehavior`?

- Single
- Multiple

Reply:
No idea, I tought about it, but I wasn't sure it'd make sense? Do you think it'd make more sense?

Reply:
Yeah, I think so, yes.

🤔 maybe better names would be

- AllowSingleMenu
- AllowMultipleMenus

in case we have more behaviors in the future.